modules/understanding-upgrade-channels.adoc

```diff
@@ -14,10 +14,112 @@ recommending the appropriate upgrade versions to your cluster. Upgrade channels
 separate upgrade strategies and also are used to control the cadence of updates.
 Channels are tied to a minor version of {product-title}. For instance,
 {product-title} {product-version} channels will never include an upgrade to a
-4.3 release. This ensures administrators make an explicit decision to upgrade to
+4.5 release. This ensures administrators make an explicit decision to upgrade to
```

Comment:
ugh.  Does this need to get bumped to 4.4 for the outgoing enterprise-4.3 branch?  @kalexand-rh , did you ask after getting a variable for the y+1 version?

Reply:
4.5 is correct in this context. Yes, I did ask @kalexand-rh about such a variable and there isn't one currently but she's looking into it.

Reply:
right, but the fact that we have 4.3 in the old text here suggests that we forgot to bump it 4.3->4.4 in the enterprise-4.3 branch.

Reply:
The KB article was written for 4.2 and this originally referenced the next minor release after 4.2  which would be 4.3. Our release is 4.4 so the next is 4.5, or am I missing something?

Reply:
This PR is fine.  4.2 is fine.  [This enterprise-4.3 tip content][1] needs to say 4.4.  I can file a PR against enterprise-4.3 when I get in.

On the other hand, we probably want to backport this PR to enterprise-4.3 before 4.3 GAs, because the KB article doesn't apply to 4.3 either.  So I'm also fine leaving the enterprise-4.3 content stale until we have time to backport this PR.  Thoughts?


[1]: https://github.com/openshift/openshift-docs/blame/19776a216514e273d5003d2a35e491d0f9a2bb22/modules/understanding-upgrade-channels.adoc#L17

Reply:
I'll do the 4.3 backport once this one has gotten the thumbs up.

Reply:
looks like @kalexand-rh bumped this in master via #18999 (fair, with GA so close), and now you need a rebase for this PR.